spec/helpers/articles_helper_spec.rb

```diff
@@ -42,5 +42,25 @@
       expect(helper.article_sanity_check).to be_falsey
     end
   end
+  
```

Comment:
the behavior is correct and the helper should probably return an empty array. Returning a string in case nothing is found can work; the specs would need to capture that expectation.